src/ipelib/flip/RSPE2B.f

```diff
@@ -854,14 +847,14 @@ SUBROUTINE TRISM1(FLDIM,IDIR,j1000S,j120S,IE,j1000N,BM,Z,JMAX,
       !.. END OF D.E. COEFFS - CONJUGATE SOLUTIONS 
       D(j1000S)=D(j1000S)-A(j1000S)*PHIUP(j1000S-1)
       D(j120S)=D(j120S)-C(j120S)*PHIUP(j120S+1)
-      if(isnan(PHIUP(j1000S-1))) write(6,*) 'GHGM isnan 1000S ',
-     >                           j1000S - 1
-      if(isnan(PHIUP(j120S+1))) write(6,*) 'GHGM isnan 120S ', j120S+1
       CALL tridiagonal_solver(FLDIM,PHIUP,j1000S,j120S,A,B,C,D,
      >                        mp,lp,2,ie)
 
       !.. PHIDWN IS EVALUATED ANALYTICALLY   ::::
-      DO J=1,JMAX-j1000N-1
+! GHGM this is wrong ?? out by 1 ....
+! replaced with the line below
```

Comment:
drop comments? seems like the off-by-one error here is connected to the adjustment above.